ReactiveObjCTests/RACSubjectSpec.m

```diff
@@ -72,14 +76,28 @@ - (void)didSubscribeWithDisposable:(RACCompoundDisposable *)disposable {
   });
 
   qck_it(@"should dispose the paired disposable when a subscription terminates", ^{
-    RACSubject* subject = [RACSubject new];
-    RACTestSubscriber* subscriber = [RACTestSubscriber new];
+    RACSubject *subject = [RACSubject new];
+    RACTestSubscriber *subscriber = [RACTestSubscriber new];
 
     [[subject subscribe:subscriber] dispose];
 
     expect(@(subscriber.disposable.disposed)).to(beTruthy());
   });
 
+  qck_it(@"should release the subscriber after subject deallocation", ^{
+    RACDisposable *subscriberDisposable;
+
+    @autoreleasepool {
+      RACTestSubscriber *subscriber = [RACTestSubscriber new];
+      RACSubject *subject = [RACSubject subject];
+      subscriberDisposable = subscriber.disposable;
+
+      [subject subscribe:subscriber];
+    }
+
+    expect(@(subscriberDisposable.disposed)).to(beTruthy());
```

Comment:
This is nice and all, but why not just hold the subscriber weakly and make sure it's nil? it's explicit, and according to the test description.

Reply:
That what happens when you copy code :)

Fixed.